.github/workflows/run_documentation_script.yaml

```
@@ -0,0 +1,37 @@
+name: Run API Documentation Generator Script
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - swagger/**
+
+jobs:  
+  run_script:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Developer Portal Definitions Repo
+        uses: actions/checkout@master
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
```

We should pin to a major version to avoid breaking changes in `fs` or `path`.
```suggestion
      - name: Setup Node.js
        uses: actions/setup-node@v3
        with:
          node_version: 16
```